### PR TITLE
refactor: Remove the old planner's interpreter creation

### PR DESCRIPTION
**Pull request by Xuanwo**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Fix https://github.com/datafuselabs/databend/issues/7757

Since this PR, all queries will be executed in the new planner.

NOTE: This PR just removed the door of the old planner. There are a lot of rooms insides.

### CODE REVIEW

File: src/query/service/src/interpreters/access/accessor.rs [hunk 1]
```diff
@@ -48,13 +48,6 @@ impl Accessor {
         Accessor { accessors }
     }
 
-    pub async fn check(&self, plan: &PlanNode) -> Result<()> {
```
Comment (BohuTANG) [inline src/query/service/src/interpreters/access/accessor.rs:51]: Also need to remove the `async fn check` from the Accessor trait.

Comment (Xuanwo) [reply src/query/service/src/interpreters/access/accessor.rs:51]: Addressed in https://github.com/datafuselabs/databend/pull/7762/commits/264a6491a1d2c57b69a4d808619dbd0d6f0d9dad

File: src/query/service/src/servers/http/clickhouse_handler.rs [hunk 2]
```diff
@@ -44,7 +44,6 @@ use serde::Serialize;
 use tracing::error;
 use tracing::info;
 
-use crate::interpreters::InterpreterFactory;
```
Comment (zhang2014) [inline src/query/service/src/servers/http/clickhouse_handler.rs:47]: Maybe rename InterpreterFactoryV2 to InterpreterFactory is better?

**Review by zhang2014 — APPROVED** [review]

**Review by sundy-li — APPROVED** [review]

**Review by BohuTANG — APPROVED** [review]

File: src/query/service/src/servers/http/clickhouse_handler.rs (hunk 2 above)
Comment (Xuanwo) [reply src/query/service/src/servers/http/clickhouse_handler.rs:47]: Addressed in https://github.com/datafuselabs/databend/pull/7762/commits/a920a22f5fac34c6ee855286b33eed8a344cc921

**Pull request merged**